src/server/generic_family_test.cc

```diff
@@ -199,6 +199,14 @@ TEST_F(GenericFamilyTest, RenameNx) {
   ASSERT_EQ(Run({"get", "y"}), x_val);
 }
 
+TEST_F(GenericFamilyTest, RenameSameName) {
+  const char kKey[] = "key";
```

Comment:
what if the key does not exist? what's the expected behavior in that case?

Reply:
That's a good comment!

Documentation doesn't specifically mention this, but a local test with a redis server shows that an error (ERR no such key) is returned. I'll modify the change to behave the same.